includes/Data/ContentBlocksResolver.php

```diff
@@ -32,13 +32,13 @@ public static function resolve_content_blocks( $node, $args, $allowed_block_name
 			// determining what fields should/should not be
 			// allowed to be returned?
 			$post    = get_post( $node->databaseId );
```

Comment:
I'm unclear why we're not just getting `$node->contentRaw`, but the existence of a comment block implies that its intentional, so I've left it alone.